src/api/l_audio.c

```diff
@@ -30,6 +24,13 @@ static int l_lovrAudioStop(lua_State* L) {
   return 0;
 }
 
+static int l_lovrAudioIsRunning(lua_State* L) {
```

Comment:
I may change to `isStarted` to match start/stop

Reply:
ooh that's much better!